### DISCUSSION TITLE
Headless Components

### DISCUSSION THREAD

**Discussion by arnau-rius** [adevinta/spark#250] [Decision making]
Let's talk about headless components

**Comment by arnau-rius** [comment]
I think it is important to be aware of what browsers we will be supporting and have this in mind when deciding (if we do) what headless component library we are going to use. I had some previous experience using Menu headless component from Radix in Milanuncios (A generalist platform from Spain), and some users complained that it was not working because they were using browsers like: https://play.google.com/store/apps/details?id=com.sec.android.app.sbrowser&hl=es&gl=US&pli=1

**Comment by acd02** [comment]
IMHO, it would be unwise for us to think that we can achieve the same level of accessibility and robust logic as Radix in such a short amount of time, and honestly, even with more time 😅

**Comment by Powerplex** [comment]
@arnau-rius I'll ask Lighthouse project supervisors if the browser list has been decided for the multi-tenant support. Maybe other streams have talked about it.

**Comment by arnau-rius** [comment]
Here is the list of browsers that apparently Radix supports: https://github.com/radix-ui/primitives/blob/8f75e5be679b9c269638bddf532a166d80edecb4/babel.config.js#L30

**Comment by Powerplex** [comment]
For Brikke this is what we had:
```
"browserslist": [
    "last 3 versions",
    "ie >= 11",
    "safari >= 4",
    "android >= 4"
  ]
```

And for Leboncoin:
```
"browserslist": [
    "cover 90% in FR",
    "last 3 versions",
    "not dead",
    "maintained node versions"
  ]
```

**Comment by arnau-rius** [comment]
Browser Support Issue:
https://github.com/adevinta/spark/issues/280

**Comment by andresz1** [comment]
There is an agreement regarding this. We are using radix and it's aligned with our browser target.
